app/models/subjects/secondary_english_mapping.rb

```diff
@@ -0,0 +1,28 @@
+# TODO: remove this bonkers logic once course mapping is done by one app!
+# The user need for this is unclear
+#
+# Does the subject list mention english, and it's mentioned in the title (or it's the only subject we know for this course)?
```

Comment:
is there a test to cover this nuget of knowledge?

Reply:
Yes, https://github.com/DFE-Digital/manage-courses-backend/blob/master/spec/services/subject_mapper_service_spec.rb#L136

Reply:
It's also covered by the production data test.

Reply:
I've removed the redundant comment in 6944134